HINWEISE.md

```diff
@@ -2,7 +2,7 @@
 
 Hier befinden sich einige Hinweise, wie verschiedene Anforderungen der HAWA umgesetzt werden können:
 
-- Metadaten (Titel, Autor(en), Matrikelnummer(n) usw.) werden in die Datei `metadaten.sty` eingetragen und dann im Dokument und in den PDF-Metadaten verwendet. Die Kommandos dazu können natürlich überall verwendet werden.
+- Metadaten (Titel, Autor(en), Matrikelnummer(n) usw.) werden in die Datei `metadaten.sty` eingetragen und dann im Dokument und in den PDF-Metadaten verwendet. Die Kommandos dazu können natürlich überall verwendet werden. Sollte ein sehr langer Titel gewählt werden, ist es möglich, dass dieser auf den Titelseiten oder den Erklärungen zu Problemen führt. Diese müssen in den jeweiligen Titelseiten- (verringerte Abstände und/oder Schriftgröße) bzw. Erklärungsdateien (zusätzliche Zeilen) behoben werden.
```

Comment:
hatten wir hierfür nicht schonmal ne lösung?

nur um sicher zu gehen, "langer titel" bezieht sich mittlerweile nicht mehr auf >1 zeile, sondern auf zu viele zeilen (--> wird zu hoch), ja?

Reply:
Dafür haben wir keine Lösung, nein. Das geht nicht automatisch zu machen... oder wenn, wüsste ich nicht, wie.

Reply:
vielleicht gehts doch --> testen

Reply:
Leider nicht (Idee war, eine feste Spaltenbreite für die zweiten Spalten in den Erklärungen zu nutzen, das bringt aber nichts)